src/chrome/content/scaffold/scaffold.js

```diff
@@ -303,6 +303,51 @@ var Scaffold = new function() {
 	});
 
 	/*
+	 * add template code
+	 */
+	this.addTemplate = Zotero.Promise.coroutine(function* (template, second) {
+		var cursorPos = _editors["code"].getSession().selection.getCursor();
+		var value = "";
+		switch(template) {
+			case "templateNewItem":
+				var outputObject = {};
+				outputObject["itemType"] = Zotero.ItemTypes.getName(second);
```

Comment:
`outputObject["itemType"]` -> `outputObject.itemType`, and same for all the assignments below